src/database/test/DatabaseTests.cpp

```diff
@@ -378,26 +382,26 @@ class SchemaUpgradeTestApplication : public TestApplication
 TEST_CASE("schema upgrade test", "[db]")
 {
     auto prepOldSchemaDB = [](SchemaUpgradeTestApplication& app,
-                              std::string const mAccountID0,
                               int64_t const mBuyingLiabilities0,
                               int64_t const mSellingLiabilities0,
-                              std::string const mAccountID1,
                               int64_t const mBuyingLiabilities1,
-                              int64_t const mSellingLiabilities1) {
+                              int64_t const mSellingLiabilities1,
+                              int64_t const mSellingLiabilities2,
+                              int64_t const mBuyingLiabilities2) {
         auto addOneOldSchemaAccount = [](SchemaUpgradeTestApplication& app,
-                                         std::string const mAccountID,
                                          int64_t const mBuyingLiabilities,
                                          int64_t const mSellingLiabilities) {
+            auto ae = LedgerTestUtils::generateValidAccountEntry();
             auto& session = app.getDatabase().getSession();
-            int64_t const mBalance = 500;
-            int64_t const mSeqNum = 7;
-            int32_t const mSubEntryNum = 0;
-            std::string const mInflationDest = "infdest";
-            std::string const mHomeDomain = "my.home";
-            std::string const mThreshold = "threshold";
-            std::string const mSigners = "autograph";
-            int32_t const mFlags = 0;
-            int32_t const mLastModified = 3;
+            std::string accountIDStr =
+                KeyUtils::toStrKey<PublicKey>(ae.accountID);
+            std::string inflationDestStr =
+                KeyUtils::toStrKey<PublicKey>(*ae.inflationDest);
+            std::string homeDomainStr;
+            homeDomainStr = decoder::encode_b64(ae.homeDomain);
```

Comment:
Nitpick: can just combine this line into the previous line. Also, these can use `auto`.

Reply:
Fixed.  Made all the declarations in that block into "auto"s, while I was at it.